tests/jvm/src/test/scala/scala/meta/tests/prettyprinters/RegressionSyntaxSuite.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.tests.prettyprinters
+
+import scala.meta.tests.parsers.ParseSuite
+
+import scala.meta._
+import munit.TestOptions
+
+class RegressionSyntaxSuite extends ParseSuite {
+
+  def check(
+      testOpts: TestOptions,
+      code: String,
+      expected: String,
+      newDialect: Option[Dialect] = None
+  )(implicit dialect: Dialect) = test(testOpts) {
+    val obtained = code.parse[Stat].get
+    val reprintedCode =
+      scala.meta.internal.prettyprinters.TreeSyntax
+        .reprint(obtained)(newDialect.getOrElse(dialects.Scala213))
+        .toString
+    assertEquals(reprintedCode, expected)
+  }
+
+  check("question-mark", "type T = List[?]", "type T = List[?]")
+  check("underscore", "type T = List[_]", "type T = List[_]")
+  check("new-dialect", "type T = List[_]", "type T = List[?]", newDialect = Some(dialects.Scala3))(
```

Comment:
you might need to have a version with scala213 (because it doesn't have significant indentation but only question mark).

Reply:
I think I missed the purpose behind your comment here, you want to test `Scala211 -> Scala213` case also?

Reply:
`questionMarkUsed` is triggered `"question-mark"` case, but sure I can also add another case

Reply:
hmm... maybe not just that. sorry, i am confused myself. perhaps actually scala3 -> scala213 is what i had in mind for this specific comment.

Reply:
Added scala211 -> scala213  test

Reply:
And also scala3 -> scala213